integration/entity-view-spring/src/main/java/com/blazebit/persistence/integration/view/spring/impl/AnnotationEntityViewConfigurationSource.java

```diff
@@ -55,9 +55,9 @@ public class AnnotationEntityViewConfigurationSource extends AbstractEntityViewC
     public AnnotationEntityViewConfigurationSource(AnnotationMetadata metadata, Class<? extends Annotation> annotation,
                                                    ResourceLoader resourceLoader, Environment environment) {
         super(environment);
-        Assert.notNull(metadata);
-        Assert.notNull(annotation);
-        Assert.notNull(resourceLoader);
+        Assert.notNull(metadata,() -> "metadata cannot be null");
+        Assert.notNull(annotation,() -> "annotation cannot be null");
+        Assert.notNull(resourceLoader,() -> "resourceLoader cannot be null");
```

Comment:
Please change to this

```suggestion
        Assert.notNull(metadata, "metadata cannot be null");
        Assert.notNull(annotation, "annotation cannot be null");
        Assert.notNull(resourceLoader, "resourceLoader cannot be null");
```

Reply:
Hi @beikov , the reason to use supplier is for the purpose of performance gain , as per documentation below supplier get triggered only if it fails, where as if we dont specify it will be initialized during loading.

```
Params:
object – the object to check messageSupplier – a supplier for the exception message to use if the assertion fails
```

Reply:
Since backward compatibility is must, accepted the code changes